enatega-multivendor-app/src/ui/hooks/useRestaurantScreenData.js

```
@@ -0,0 +1,314 @@
+      console.error(`Error reading fragment for Food:${itemId}`, error)
+      return null
+    }
+  }
+
+  const dataList =
+    popularItems &&
+    popularItems?.popularItems?.map((item) => {
+      const foodDetails = fetchFoodDetails(item?.id)
+      return foodDetails
+    })
+
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)